From 2940d8e5c74faa5dcd5279c8166064ade3da5695 Mon Sep 17 00:00:00 2001
From: Bernd Waibel <waebbl@gmail.com>
Date: Sun, 17 May 2020 21:13:44 +0200
Subject: [PATCH] [Gentoo-specific] Disable ccache building

On Gentoo we use a USE flag to determine whether to use
ccache for building or not.

Signed-off-by: Bernd Waibel <waebbl@gmail.com>
---
 openscad.pro | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/openscad.pro b/openscad.pro
index 2d190dd..9c4e67b 100755
--- a/openscad.pro
+++ b/openscad.pro
@@ -214,10 +214,10 @@ mdi {
   DEFINES += ENABLE_MDI
 }
 
-system("ccache -V >/dev/null 2>/dev/null") {
-  CONFIG += ccache
-  message("Using ccache")
-}
+#system("ccache -V >/dev/null 2>/dev/null") {
+#  CONFIG += ccache
+#  message("Using ccache")
+#}
 
 include(common.pri)
 
-- 
2.26.2

